wiki/History_of_osu!/de.md

```diff
@@ -1,25 +1,28 @@
 ---
 outdated: true
-outdated_translation: true
-outdated_since: 1ef63c7a78e0aed007a64cef508872abf9c5c123
 ---
 
 # Geschichte von osu!
 
 Aufzeichnungen der Geschichte von osu!, alle im osu!-Wiki.
 
-- [2007](2007) - Anfängliche Spielentwicklung
-- [2008](2008) - Taiko und Catch the Beat, mehr Mods und wesentliche Verbesserungen
-- [2009](2009) - osu! iPhone
-- 2010
-- [2011](2011) - osu!-Wiki
-- [2012](2012) - Fünf Jahre osu!, osu!mania, Breitbildunterstützung und visuelle Einstellungen
-- [2013](2013) - Überarbeitung der UI, HD-Skins, verbesserte Breitbildunterstützung, osu!mania-Editor, Sammlungen im Spiel, moddingV2 wurde vorgestellt, Disqus, osu!api, Overlays für Genauigkeit, Comic Fiesta 2013, unbegrenzte Downloads und osu!cuttingedge (nur osu!supporters)
-- [2014](2014) - BPM-Scrolling, Intro-/Outro-Sequenzen im Spiel, osu!cuttingedge (für alle) und Schnee!
-- [2015](2015) - jizz (neu gestaltetes osu!web), osu!weekly und osu!coins
-- [2016](2016) - scorev2, Ankündigung einer neuen quelloffenen Version von osu! namens "lazer" (vorher bekannt als "osu!lazer") und eine Überarbeitung des osu!-Wikis
-- [2017](2017) - erste Veröffentlichung des Builds lazer von osu! und Discord-Integration auf dem Stable-Client
-- [2018](2018)–Heute - ?
+| Jahr | Beschreibung |
+| :-: | :-- |
+| [2007](2007) | Anfängliche Spielentwicklung |
+| [2008](2008) | osu!taiko, osu!catch, mehr Mods und wesentliche Verbesserungen |
+| [2009](2009) | osu! iPhone |
+| 2010 | *Undokumentiert* |
+| [2011](2011) | osu!-Wiki |
+| [2012](2012) | Fünf Jahre osu!, osu!mania, Breitbildunterstützung und visuelle Einstellungen |
+| [2013](2013) | Überarbeitung der UI, HD-Skins, verbesserte Breitbildunterstützung, osu!mania-Editor, Sammlungen im Spiel, moddingV2 wurde vorgestellt, Disqus, osu!api, Overlays für Genauigkeit, Comic Fiesta 2013, unbegrenzte Downloads und osu!cuttingedge (nur für osu!supporter) |
+| [2014](2014) | osu!mania BPM-Scrolling, Intro-/Outro-Sequenzen im Spiel, osu!cuttingedge (für alle) |
+| [2015](2015) | osu!web-Neugestaltung, osu!weekly und osu!coins |
+| [2016](2016) | scorev2, Ankündigung einer neuen quelloffenen Version von osu! namens "lazer" (vorher bekannt als "osu!lazer") und eine Überarbeitung des osu!-Wikis |
+| [2017](2017) | Erste Veröffentlichung des Builds lazer von osu! und Discord-Integration auf dem Stable-Client |
+| [2018](2018) | osu!next, deprecation der alten Webseite, Anzeige der Gesamtspielzeit |
```

Comment:
```suggestion
| [2018](2018) | osu!next, Stilllegung der alten Webseite, Anzeige der Gesamtspielzeit |
```
Maybe this could work, can't really think of any translations that don't indicate that the old website has shut down.

Reply:
That's a great idea. I like it!